config/src/network_id.rs

```diff
@@ -108,16 +165,12 @@ impl PartialOrd for NetworkId {
             match self {
                 NetworkId::Validator => Ordering::Less,
                 NetworkId::Public => Ordering::Greater,
-                NetworkId::Private(_) => match other {
+                NetworkId::Invalid => Ordering::Less,
+                NetworkId::Vfn => match other {
                     NetworkId::Validator => Ordering::Greater,
                     NetworkId::Public => Ordering::Less,
-                    NetworkId::Private(_) => {
-                        if self.is_vfn_network() {
-                            Ordering::Less
-                        } else {
-                            Ordering::Greater
-                        }
-                    }
```

Comment:
I think we can just derive `Eq, PartialEq, Ord, PartialOrd` now

Reply:
also won't need the `Ord` impl above

Reply:
I'd like to leave this implementation of ord, VFN is greater than public, but it's not in that order

Reply:
I think you should be able to put this impl in `Ord` and remove the panic?

Reply:
Yeah, I just reordered it and we can cut that